core/src/i18n/en.yaml

```
@@ -226,6 +230,12 @@ clusters:
       sessionStorage: sessionStorage
       unknown: Unknown
     title: Storage Type
+  wizard:
+    kubeconfig: Choose configuration
+    update: Update configuration
+    storage: Storage mode
+    info: We don't store the kubeconfig file on server, it remains in the browser.
+    incomplete: It looks like your kubeconfig is incomplete. Please fill the additional fields.
```

```suggestion
    incomplete: It looks like your kubeconfig is incomplete. Fill out the additional fields.
```
No "please" needed :)